### PR TITLE
fixes most doozer.io build targets...

### PR DESCRIPTION
**Pull request by mpmc**
Some are completely broken due to a change in how GitHub/git handle the 'git' protocol and SSL.

### CODE REVIEW

File: support/getmuxlist [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # Arguments
 [ -z "$BRANCH" ] && BRANCH=tvheadend
 URL_SCHEME=https
-[ ! -z "$DOOZER_JOBID" ] && URL_SCHEME=git
+[ ! -z "$DOOZER_JOBID" ] && URL_SCHEME=https
```
Comment (Flole998) [inline support/getmuxlist:9]: Why not remove this stuff completely and add it to the line below?

**Comment by Flole998** [comment]
Why are there builds failing again? I thought I had all of them working before?

**Comment by mpmc** [comment]
@Flole998 I did min changes possible.

I think if you can correct/update the ssl certs in the env they'll build again. Another option would be just to replace all that logic and download a tar.gz from github over http instead.

File: Autobuild/jammy-i386.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+AUTOBUILD_CONFIGURE_EXTRA="${AUTOBUILD_CONFIGURE_EXTRA:-} --arch=i686"
+DEBDIST=bionic
```
[the remark this answers is not in the document]
Comment (mpmc) [reply Autobuild/jammy-i386.sh:2]: Good catch. It won't build anyway (no docker image for i386 jammy). I'll correct.

**Comment by mpmc** [comment]
@Flole998 See https://github.blog/2021-09-01-improving-git-protocol-security-github/

**Comment by Flole998** [comment]
Aren't the new Ubuntu releases amd64-only? I tink for focal we also only have amd64?

**Comment by mpmc** [comment]
> Aren't the new Ubuntu releases amd64-only? I think for focal we also only have amd64?

Kinda, apparently they're still fading i386 out. Won't hurt to leave it I suppose?

**Comment by Flole998** [comment]
We're moving away from doozer, I applied the getmux-thing manually but all the other changes will soon be obsolete anyways.

**Pull request closed without merging**